src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -31,13 +31,14 @@ public function get( $node_id, $element ) {
 		foreach ( $this->nodes_to_translate as $node_type => $node_data ) {
 			if ( $this->conditions_ok( $node_data, $element ) ) {
 				foreach ( $node_data['fields'] as $key => $field ) {
-					$field_key    = $field['field'];
-					$string_value = null;
+					$field_key     = $field['field'];
+					$field_wrapper = isset( $field['field_wrapper'] ) ? $field['field_wrapper'] : $key;
```

Comment:
@strategio What does `field_wrapper` really mean? Can you give an example? I suspect it's a bad name for it. I was also confused because Igor added `wrappers` for something else and I thought this was the same.

Reply:
@BruceOnTheGo, basically we have 2 cases to handle:

1. String as a flat element:

```php
[
    'settings' => [
        'some_field' => 'This is a string to translate',
    ],
];
```

2. String "wrapped" inside an array:

```php
[
    'settings' => [
        'some_field' => [
            'something' => 'This is a string to translate',
        ],
    ],
];
```

As you can see in 2nd case, "some_field" could be considered as a wrapper of the actual field we want to translate "something".

Now, I understand your concern about the naming, this is something I missed... My inspiration is stuck, maybe we can call it `field_array`... Do you have some ideas?

Reply:
@strategio How will the `$field` variable look for both these cases?

Reply:
1. Flat field

```php
[
    'field'       => 'some_field'
    'type'        => __( 'Some Field', 'sitepress' ),
    'editor_type' => 'LINE'
]
```

2. Array field

```php
[
    'field_wrapper' => 'some_field',
    'field'         => 'something',
    'type'          => __( 'Some Field', 'sitepress' ),
    'editor_type'   => 'LINE'
]
```

Note that in case 2, we will also keep the ability to use the array key as the "wrapper" for backward compatibility. We use it for "links" like this:

```php
'link' => [
	'field'       => 'url',
	'type'        => __( 'Image: Link URL', 'sitepress' ),
	'editor_type' => 'LINK'
]
```

Reply:
So why don't we do this?
```php
'some_field' => [
    'field'         => 'something',
    'type'          => __( 'Some Field', 'sitepress' ),
    'editor_type'   => 'LINE'
]
```